openmm_septop/septop_analysis.ipynb

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.            writer.writerow(["ligand", "DG(i-&gt;j) (kcal/mol)",
</pre><p>Should be DDG?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/225/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>